File: GestionInterim/app/src/main/java/com/example/gestioninterim/LoginActivity.java
package com.example.gestioninterim;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class LoginActivity extends AppCompatActivity {

    TextView toConnect;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        toConnect = findViewById(R.id.clickHere);

        toConnect.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(LoginActivity.this, InscriptionActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: GestionInterim/app/src/main/java/com/example/gestioninterim/LoginActivity.kt
package com.example.gestioninterim

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        var toConnect:TextView = findViewById<TextView>(R.id.clickHere)
        toConnect.setOnClickListener(View.OnClickListener {
            val intent = Intent(this@LoginActivity, InscriptionActivity::class.java)
            startActivity(intent)
        })
    }
}